homeassistant/components/srp_energy/const.py

```diff
@@ -11,3 +11,12 @@
 
 PHOENIX_TIME_ZONE = "America/Phoenix"
 MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=1440)
+
+SENSOR_NAME = "Energy Usage"
+SENSOR_TYPE = "usage"
+
+DEVICE_CONFIG_URL = "https://www.srpnet.com/"
+DEVICE_MANUFACTURER = "srpnet.com"
+DEVICE_MODEL = "Service Api"
+DEVICE_NAME_ENERGY = "Energy consumption"
+DEVICE_NAME_PRICE = "Energy consumption price"
```

Comment:
Only me or I don't see they are used anywhere?

Reply:
I've added these back to the device setup.